grumphp.yml

```diff
@@ -39,6 +39,16 @@ parameters:
 
   # Psalm
   tasks.psalm.blocking: true
+  tasks.psalm.ignore_patterns:
+      - "/.github/"
+      - "/.idea/"
+      - "/build/"
+      - "/benchmarks/"
+      - "/node_modules/"
+      - "/resource/"
+      - "/spec/"
+      - "/var/"
+      - "/vendor/"
```

Comment:
I realised in CI Psalm wasn't running in `tests` where the static analysis checks are, so removing that ignore pattern here. I was still running them on local 😄